requirements.txt

```diff
@@ -1,3 +1,4 @@
 ipython
 pyyaml
 coloredlogs
+git+https://github.com/slaclab/happi#egg=happi
```

Comment:
Same as before, is this an explicit requirement? i.e. can I load the module in some capacity if I don't have `happi` installed? If not, it doesn't belong here.

Reply:
You are correct. I'll change this.